### ISSUE TITLE
Add docs for logging

### ISSUE THREAD

**Issue by ledex** [spaceuptech/space-docs#12]
When [space-cloud/pull/508](https://github.com/spaceuptech/space-cloud/pull/508) is merged, docs for the logging should be added.

**Comment by Jayesh333** [comment]
Yeah @ledex, we would also have to figure out what all messages need to be logged at what level.

**Comment by ledex** [comment]
Yes @Jayesh333. You can find the provided log-levels in [utils/loglevel.go](https://github.com/spaceuptech/space-cloud/pull/508/files#diff-df99cc73a2e2f0e7f8da2c3f19e288e4). Currently there are DEBUG, INFO, WARNING and ERROR.

**Comment by Jayesh333** [comment]
Yeah I will have a look at it @ledex.
